src/app/components/Nav/Nav.js

```diff
@@ -10,9 +10,11 @@ import "./Nav.scss";
 const Nav = () => {
 	const { pathname } = useLocation();
 	const [mobileExpand, setMobileExpand] = useState(false);
+
 	const toggleMobileExpand = () => {
 		setMobileExpand(!mobileExpand);
 	};
```

Comment:
I know you didn't write this but can you wrap in usecallback and then use the functional set state

Reply:
👀  i wrapped it in a callback, but i got no clue what "functional set state" is referring to 😅